code/modules/mob/living/simple_animal/friendly/drone/verbs.dm

```diff
@@ -12,6 +12,7 @@
 	to_chat(src, "<b>Drone Laws</b>")
 	to_chat(src, laws)
 
+/* Removed in favor of night vision
```

Comment:
Just remove everything that's commented out, we can see it in the git history anyway.

Reply:
alright cool, I didn't know that